### PR TITLE
Feat/interest view

### PR DESCRIPTION
**Pull request by mostarz93**
Added a view func for current accrued interest

### CODE REVIEW

File: contracts/tests/LineLends.t.sol [hunk 1]
```diff
@@ -0,0 +1,137 @@
+pragma solidity ^0.8.9;
```
Comment (kibagateaux) [inline contracts/tests/LineLends.t.sol:1]: "Lens" not "Lends" lol

File: contracts/tests/LineLends.t.sol [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+        assertEq(interestAccrued, 0);
+
+        uint256 getInterest = line.interestAccrued(id);
+
+        assertGt(getInterest, 0);
+
+    }
+
+    function test_interest_accrued_vs_interest_viewed_with_time() public {
```
Comment (kibagateaux) [inline contracts/tests/LineLends.t.sol:106]: Can u add one more test where accrue interest and then warp again and then check `position.interestAccrued == oldInterest` and  `initerestAccrued() == position.interestAccrued + interestRate.getAccruedInterest()`.

Can you also add one test that calls the interest rate contract directly to check that interestViewed equals what interest rate returns

File: contracts/tests/LineLends.t.sol [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+        vm.warp(30 days);
+
+        (,,uint256 interestAccrued,,,,,) = line.credits(id);
+
+        assertEq(interestAccrued, 0);
+
+        uint256 getInterest = line.interestAccrued(id);
+
+        assertGt(getInterest, 0);
```
Comment (kibagateaux) [inline contracts/tests/LineLends.t.sol]: `assertEq(getInterest, interestRate.getInterestAccrued(id, principal, deposit)`

File: contracts/tests/LineLends.t.sol (hunk 1 above)
Comment (mostarz93) [reply contracts/tests/LineLends.t.sol]: done

**Pull request merged**